src/app/auth/register/page.tsx

```diff
@@ -9,7 +9,6 @@ import { State, register } from "@/app/auth/register/actions";
 
 const Register: NextPage = () => {
 	const [state, formAction] = useFormState<State, FormData>(register, {});
-
```

Comment:
Ehhhh leave the blank line there. I like having some separation between hooks and the actual React elements.

Reply:
Of course!